apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -52,6 +52,51 @@ function load_full_site_editing() {
 }
 add_action( 'plugins_loaded', __NAMESPACE__ . '\load_full_site_editing' );
 
+/**
+ * Whether or not FSE is active.
+ * If false, FSE functionality can be disabled.
+ *
+ * @returns bool True if FSE is active, false otherwise.
+ */
+function is_full_site_editing_active() {
+	$supported_themes = [ 'maywood' ];
```

Comment:
I'm sure we'll immediately refactor this but let's use a constant outside of this function

```
const SUPPORTED_THEMES = [ 'maywood' ];
```